presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -28,8 +28,8 @@
 import com.facebook.presto.spi.type.Type;
 import com.facebook.presto.spi.type.VarcharType;
 import com.google.common.collect.ImmutableSet;
+import com.mysql.cj.jdbc.JdbcStatement;
 import com.mysql.jdbc.Driver;
```

Comment:
should this be also changed to `com.mysql.cj.jdbc.Driver` according to https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-api-changes.html